tests/receivers/smartagent/postgresql/testdata/client/requests.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Does this file define what metrics we are going to observe as specified in all.yaml? If so can we explain what is going on here?

Reply:
Yes, it's responsible for populating most of the values for all generated metrics.  Updated to provide more info.